packages/arb-token-bridge-ui/src/util/index.ts

```diff
@@ -53,3 +53,28 @@ export const getAPIBaseUrl = () => {
   // Resolves: next-js-error-only-absolute-urls-are-supported in test:ci
   return process.env.NODE_ENV === 'test' ? 'http://localhost:3000' : ''
 }
+
+type FeatureFlag = 'batch'
+
+export const isExperimentalFeatureEnabled = (flag: FeatureFlag) => {
+  const query = new URLSearchParams(window.location.search)
+  const featureFlags = query.get('experiments')
+
+  if (!featureFlags) {
+    return false
+  }
+
+  if (!flag) {
+    // we only want to check if any feature flag is enabled
+    return true
+  }
+
+  return featureFlags.split(',').includes(flag)
+}
+
+export const isExperimentalModeEnabled = () => {
+  const query = new URLSearchParams(window.location.search)
+  const featureFlags = query.get('experiments')
+
+  return featureFlags !== null
```

Comment:
this currently doesn't validate if any actual feature flag was provided, so it would return `true` for `experiments=bla`